index.bs

```diff
@@ -638,6 +592,60 @@ Note: {{XRReferenceSpaceType/"local"}} is always included in the [=requested fea
 
 Note: For example, several VR devices support either configuring a safe boundary for the user to move around within or skipping boundary configuration and operating in a mode where the user is expected to stand in place. Such a device can be considered to be [=capable of supporting=] {{"bounded-floor"}} {{XRReferenceSpace}}s even if they are currently not configured with safety boundaries, because it's expected that the user could configure the device appropriately if the experience required it. This is to allow user agents to avoid fully initializing the [=XRSession/XR device=] or waiting for the user's environment to be recognized prior to [=resolve the requested features|resolving the requested features=] if desired. If, however, the user agent knows that the boundary state at the time the session is requested without additional initialization it may choose to reject the {{"bounded-floor"}} feature if the safety boundary not already configured.
 
+
+Fingerprinting considerations of {{XRSystem/isSessionSupported()}} {#issessionsupported-fingerprinting}
+----------------------------------
+Because {{XRSystem/isSessionSupported()}} can be called without user activation it may be used as a fingerprinting vector despite the limited amount of information it reports.
+
+<dfn enum-value for="PermissionName">"xr-session-supported"</dfn> [=powerful feature=] gates access to the {{XRSystem/isSessionSupported()}} API.
+
+The {{PermissionName/"xr-session-supported"}}’s permission-related algorithms and types are defined as follows:
+
+
+<dl>
+<dt>[=permission descriptor type=]</dt>
+<dd>
+
+<pre class="idl">
+dictionary XRSessionSupportedPermissionDescriptor: PermissionDescriptor {
+  XRSessionMode mode;
+};
+</pre>
+
+{{PermissionDescriptor/name}} for {{XRPermissionDescriptor}} is {{PermissionName/"xr-session-supported"}}.
+
+</dd></dl>
+
+### Automatic granting of {{PermissionName/"xr-session-supported"}} ### {#automatic-granting-xr-session-supported}
+
+<section class="non-normative">
+
+In many situations, {{XRSystem/isSessionSupported()}} is not actually a fingerprinting vector. For such systems it is useful to automatically grant the {{PermissionName/"xr-session-supported"}} preference to provide for a better user experience and avoid permissions fatigue.
```

Comment:
not a substantive concern / note, but its odd and awkward to start any text (normative or non-normative) with a negative statement.  Can this be rephrased to state what the section is about, and not what the section is not about :)

So, straw suggestion:

There is often tension between privacy and personalization on the Web.  This section provides guidance on where that tradeoff can be circumscribed, and when the user agent can describe the browser's WebXR capabilities to a site without any privacy reduction.